hyper/http11/connection.py

```diff
@@ -396,3 +427,23 @@ def __enter__(self):
     def __exit__(self, type, value, tb):
         self.close()
         return False  # Never swallow exceptions.
+
+
+def _create_tunnel(proxy_host, proxy_port, target_host, target_port,
```

Comment:
As a minor coding style note, it's better to put functions like these towards the *top* of the file rather than towards the bottom.